### DISCUSSION TITLE
Message to the collaborators

### DISCUSSION THREAD

**Discussion by jtr13** [jtr13/cc21fall2#2] [Announcements]
Welcome! As soon as we have some pull requests, I will schedule a meeting to discuss the merge process. From now on, all discussion should happen here rather than in CourseWorks.

**Comment by jtr13** [comment]
Please check off your name (and edit if you like to be called by a different name) so I know you're getting email notifications of mentions. If you're not able to do so, it means you didn't accept the invitation to be a collaborator on the repo.

- [x] Abhimanyu  @Abhimanyu210100 
- [x] Adrian @AdrianGarciaH
- [x] Alex @alexnsaunders 
- [x] Alexandria @alexandriaGuo
- [x] Gangadhara Reddy @Gangadharareddy514
- [x] Joy @joywang1999
- [x] Kevia @kq2153
- [x] Lea @BenyaminLea
- [x] Nicolo @nicric99
- [x] Raihan @raihan-bashir
- [x] Ridwan @rao53
- [x] Sarthak @sarthak-arora1997
- [x] Shuqing @pyropig
- [x] Tyler @Tylermarshall21
- [x] Yuan @HannahFromAland
- [x] Zhenyu @Francis-ZY-Yuan

**Comment by jtr13** [comment]
Hi All,

I've scheduled a virtual meeting tomorrow to go over your jobs as PR mergers. I will meet from 5:00-5:30 with those who signed up for special roles (assigning PRs, managing the _bookdown.yml file) and then from 5:30-6:00 with everyone. It will be recorded if you can't make it, but hoping you can attend live (on Zoom). Please see Zoom class sessions in CourseWorks for the link as I don't want to publish it publicly.

Joyce

**Comment by jtr13** [comment]
Hi All,

If you weren't able to attend today's meeting, please watch the recording which is available in CourseWorks in the usual place for class recordings. Even though only the second half of the meeting was meant for all the PR mergers, I actually think the first half will be helpful for everyone, so in short I encourage to watch the whole session.

Let me know if you have any questions.

Joyce

**Comment by alexnsaunders** [comment]
I completed my assignments - should we wait for assignment or can we pick up unassigned ones? I tend to have random blocks of time when I can work on so sometimes I can do more but other times I'm not available - (weekends tough as have a family)

**Comment by jtr13** [comment]
I hear you… @Gangadharareddy514 @Abhimanyu210100  would it work for Alex to assign himself some PRs. Or maybe there’s another way to coordinate so he can work when he has a random block of time?

**Comment by Abhimanyu210100** [comment]
@jtr13 @alexnsaunders I will assign more PRs right now will try to finish assigning everything over the next couple of days

**Comment by jtr13** [comment]
It only takes a few minutes to assign the PRs... I would really like to see it happen today so that the PR mergers can get to work over the weekend. If you're not able to do it is there someone else who can?

**Comment by jtr13** [comment]
I see I spoke a little too soon as most of them are assigned. Still let's get them all assigned to today to keep things moving.

**Comment by jtr13** [comment]
Hi all! 
Please keep an eye on the builds (https://github.com/jtr13/cc21fall2/actions) -- whether it's a PR you merged or not -- and tag me on the closed PR that is causing the problem. I have fixed all the previous problems and the build is now working, doubling our rendered chapters to 68!  Also, if you choose to fix the problem yourself and it involves removing the chapter or setting all the code chunks to `eval=FALSE` please either open a new issue or comment on an open one that relates to the same problem. The two main ones are Python code not working and calls to APIs without a key.

**Comment by joywang1999** [comment]
Do we need to take off `changes requested` label after merging?

**Reply by alexnsaunders** [reply]
I don't know but I don't think so. It doesn't make a difference to the merging/bookrun process and I would think, looking ahead, it might be nice to know how many ppl were able to follow the instructions and submit correctly first time vs not.

Another quick q - now we are running I'm sharing the page in the comments. Where we see things that are amiss, but not incorrect for the submission, typos or links that don't work, I'm suggesting changes in the comments - I presume a_ that's ok! and b)they would need to submit a new PR correct? in which case you can assign back to me.

**Comment by jtr13** [comment]
The build is working again ✅ 😄 
Remember: 
* Clear commit messages
* Open an issue if the build is failing... it's much easier to fix as things come up rather than have the problems pile up
Great work team... 96 chapters and counting! 😮
